### ISSUE TITLE
Retroactively edit a Voronoi pattern

### ISSUE THREAD

**Issue by caroliner99** [hanskellner/Fusion360Voronoi#15]
I'd love to be able to go back and edit a Voronoi pattern. Integration into the feature timeline would be great, so that I could go back and change the number of cells or space between cells in the pattern without having to make a whole new pattern and delete the old one.

**Comment by hanskellner** [comment]
@caroliner99 Yes, that would be a nice addition.  It's something I've considered in a rewrite I've been planning.  I'll star it in my todo list for the update.  Thanks for the suggestion.

**Comment by coyotespike** [comment]
Seconding this, should you ever have the time! Wonderful tool, thanks for making. The ability to edit a Voronoi pattern would be basically the same as parameterization - close enough anyway!
